third_party/2/enum.pyi

```diff
@@ -16,7 +16,6 @@ class Enum(metaclass=EnumMeta):
     name = ...  # type: str
     value = ...  # type: Any
 
-class IntEnum(int, Enum):  # type: ignore
-    def __new__(cls: Type[_T], value: Any) -> _T: ...
+class IntEnum(int, Enum): ...
```

Comment:
If you're removing the explicit `__new__`, would it make sense to make the base class order "Enum, int" so that `__new__` is inherited from `Enum`, not `int`?